data/tools/networking.json

```diff
@@ -609,6 +609,14 @@
       "price": "Free",
       "online": "False"
     },
+    { 
+      "name": "Xprobe2",
+      "source": "https://github.com/binarytrails/xprobe2.git", 
+      "description": "Xprobe2 is a remote active operating system fingerprinting tool.",
```

Comment:
See [this](https://inventory.raw.pm/docs/docs/en/add/) and [this](https://inventory.raw.pm/docs/docs/en/format/)

- No final dot
- No tool name